.github/scripts/build-benchmark-docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -euo pipefail
+
+TODAY=$(date +"%Y%m%d")
+
+# Get the latest pytorch docker image
+TORCH_LATEST=$(docker images | grep "pytorch\s" | head -n 1)
```

Comment:
this is using the pytorch image you built in 'build-docker.sh'? 

i wonder why you build pytorch docker.  is it bc it already provides cuda or other deps?

if you just need cuda/deps but don't need pytorch in the base docker (bc you can install pytorch from conda or pip), then maybe it's easier to skip building pytorch and instead put 'from X' in your benchmark DOCKERFILE, where X is either an nvidia public docker image, or one of the pytorch docker images already in our docker hub?

Reply:
I use pytorch docker because it packs nightly pytorch, torchtext, torchvision (from conda), as well as all their dependencies (including nvidia-cuda, cudnn, and so on). Therefore, I don't need to concern about any version or dependency problems, therefore avoids maintenance overhead (which is place on the pytorch dockerfile).
I believe it is still worthwhile to continue using it as a base image, if we need to customize pytorch version (not using nightly), we could first uninstall from conda, then install using pip.

Reply:
i'm not sure what maintennence or dependency you mean.  if you just type 'conda install -c pytorch-nightly pytorch torchtext torchvision' doesn't that take care of deps?  

anyway, your approach seems ok just thought it might be more work than necessary

Reply:
Just for record, currently pytorch doesn't build a nightly docker image so we have to build it from its Dockerfile. In the Dockerfile, pytorch team maintains all the dependencies such as cuda, cudnn, among other packages. We should rely on the correctness of the official Dockerfile instead of managing dependencies of our own.